qi-test/tests/compiler/rules/full-cycle.rkt

```diff
@@ -21,12 +22,17 @@
 
   ;; A macro that accepts surface syntax, expands it, and then applies the
   ;; indicated optimization passes.
-  (define-syntax-parser test-compile~>
+  (define-syntax-parser test-passes~>
     [(_ stx)
      #'(expand-flow stx)]
     [(_ stx pass ... passN)
      #'(passN
-        (test-compile~> stx pass ...))]))
+        (test-passes~> stx pass ...))])
```

Comment:
I think this macro is no longer being used after being removed from the normalize -> deforest test below.